apps/web/public/static/locales/en/common.json

```diff
@@ -2613,5 +2613,7 @@
   "limit_team_booking_frequency_description": "Limit how many times members can be booked across all team event types (collective and round-robin)",
   "booking_limits_updated_successfully": "Booking limits updated successfully",
   "you_are_unauthorized_to_make_this_change_to_the_booking": "You are unauthorized to make this change to the booking",
+  "hide_calendar_event_details": "Hide calendar event details",
```

Comment:
Can we try to make this more descriptive as well?
'Make calendar event private'
'Hide calendar event details on shared calendars'